### ISSUE TITLE
There seems to be a lack of description of conditional-access sample

### ISSUE THREAD

**Issue by shmiki** [azure-ad-b2c/samples#260]
I encountered the following error when I tried the following sample policy.

Target sample policy: https://github.com/azure-ad-b2c/samples/blob/master/policies/conditional-access/policy/TrustFrameworkExtensionsCA.xml

error message: Output claim type "estsRequestId" specified in the technical profile with id "SelfAsserted-LocalAccountSignin-Email" in policy "B2C_1A_SIGNUP_SIGNIN" of tenant "xxxxx.onmicrosoft.com" does not specify a UserInputType or a DefaultValue, and is not retrieved from a ValidationTechnicalProfile either.
![image](https://user-images.githubusercontent.com/67548423/126178129-ad292d85-cc60-4b0f-92b4-b519026e7846.png)

According to the error message, it seems that SelfAsserted-LocalAccountSignin-Email needs a default value for Output claim "estsRequestId".
So I set the default value as follows, and the error was resolved.

OutputClaim ClaimTypeReferenceId="estsRequestId" DefaultValue="00000000-0000-0000-0000-000000000000"

I am thinking that the DefaultValue is missing in the sample policy.
If I'm right, could you please update the sample policy?

**Comment by JasSuri** [comment]
Does appear something is missing there, looking into it.

**Comment by shmiki** [comment]
Thank you for your reply.
Sorry. I must have been mistaken.
I made on this policy,  TrustFrameworkExtensionsCA.xml based on active-directory-b2c-custom-policy-starterpack/SocialAndLocalAccounts/TrustFrameworkBase.xml

But readme.md says it is based on the policy active-directory-b2c-custom-policy-starterpack/SocialAndLocalAccounts/TrustFrameworkExtensions.xml.
![image](https://user-images.githubusercontent.com/67548423/126294969-550cf9f0-eaff-49c5-8864-c8f4361f1bb7.png)

I actually recreated the policy according to reamdme.md, and the error described above did not occur.
